### PR TITLE
add meal-sharing week2 files

### PR DESCRIPTION
**Pull request by polyovik**
meal-sharing week2 homework

### CODE REVIEW

File: src/backend/api/services/reservations.services.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+const selectAllReservationsQuery = () => {
+  return knex('reservations').select('*');
+};
+
+const addNewReservationQuery = (newReservationData) => {
+  return knex('reservations').insert(newReservationData).into('reservations');
+};
+
+const getReservationById = (reservationId) => {
+  return knex('reservations').select('*').where({ id: reservationId });
```
Comment (itsgauravsaxena) [inline src/backend/api/services/reservations.services.js:12]: May be select is not needed here? It should work without select as well

File: src/backend/api/services/meals.services.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const knex = require('../../database');
+
+const selectAllMealsQuery = () => {
+  return knex('meals').select('*');
```
Comment (itsgauravsaxena) [inline src/backend/api/services/meals.services.js:4]: It should work without select(*)

**Review by itsgauravsaxena — COMMENTED** [review]
Just a minor comment

File: src/backend/api/services/meals.services.js (hunk 2 above)
Comment (polyovik) [reply src/backend/api/services/meals.services.js:4]: i will pay attention to that. thanks for your time!

**Pull request merged**